### PR TITLE
Tidy ola-client code

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

**Comment by nomis52** [comment]
I'm not sure about changing the DisplayXXX functions. We're losing a lot of the horizontal widget now.

**Comment by peternewman** [comment]
I've generally/always moved the newlines, so the output is the same, it's just the code formatting has changed.

File: examples/ola-client.cpp [hunk 1]
```diff
@@ -651,16 +662,15 @@ void DisplayUniverseNameHelp(const options &opts) {
...
-  "Change the merge mode for the specified universe. Without --ltp it will\n"
-  "revert to HTP mode.\n"
-  "\n"
-  "  -h, --help                Display this help message and exit.\n"
-  "  -l, --ltp                 Change to ltp mode.\n"
-  "  -u, --universe <universe> Id of the universe to change.\n"
-  << endl;
+  cout << "Usage: " << opts.cmd << " --universe <uni> [ --ltp]\n"
+          "\n"
+          "Change the merge mode for the specified universe. Without --ltp\n"
```
Comment (nomis52) [inline examples/ola-client.cpp]: This is the type of thing I'm not a fan of, the line length is down to 64 characters.

File: examples/ola-client.cpp [hunk 2]
```diff
@@ -669,33 +679,33 @@ void DisplayUniverseMergeHelp(const options &opts) {
...
-  "  -d, --device <device>    Id of device to set priority for.\n"
-  "  -h, --help               Display this help message and exit.\n"
-  "  -i, --input              Set an input port\n"
-  "  -o, --override <value>   Set the port priority to a static value.\n"
-  "  -p, --port <port>        Id of the port to set priority for.\n"
-  << endl;
+  cout << "Usage: " << opts.cmd
+       << " --device <dev> --port <port> [--override <value>]\n"
+          "\n"
+          "Set a port's priority, without the --override flag this will set\n"
```
Comment (nomis52) [inline examples/ola-client.cpp]: Here as well

File: examples/ola-client.cpp (hunk 1 above)
Comment (peternewman) [inline examples/ola-client.cpp]: Fixed.

File: examples/ola-client.cpp (hunk 2 above)
Comment (peternewman) [inline examples/ola-client.cpp]: Fixed.

**Pull request merged**